kolibri/deployment/default/settings/base.py

```diff
@@ -190,6 +190,21 @@
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
+# File Storage Backend
+# https://docs.djangoproject.com/en/3.2/ref/files/storage/
+
+if not os.environ.get("DEFAULT_FILE_STORAGE"):
```

Comment:
What's the extra environment variable here for? Can we not just gate this on the conf.OPTIONS?

Reply:
This is still unresolved, I'm not seeing any need for this extra check.